README.md

```diff
@@ -301,11 +306,57 @@ updateNavigationBar({
     expandedTitle: 'Hello, World!',
     showBackButton: true,
     showReloadButton: false,
-    showProfileButton: false,
-    backgroundColor: '#FF0000', // red
+    backgroundColor: '#FF0000',
+    leftNavigationIcons: [
+        {
+            name: 'icon name',
+            iconEnum: 'SOME_ICON',
+            badge: {
+                show: true,
+                nativeLogic: 'INBOX',
+            },
+        },
+    ],
+    rightNavigationIcons: [
+        {
+            name: 'icon name',
+            iconEnum: 'icon enum value',
+            icon: {
+                url: 'https://path/to/icon',
+                urlDark: 'https://path/to/icon/dark',
+            },
+            badge: {
+                show: true,
+                number: 1,
+            },
+        },
+    ],
+    resetToDefaultState: true,
+    trackingProperties?: {'name': 'some icon clicked'},
 });
 ```
 
+### onNavigationBarIconClicked
+
+Listen to navigation bar icon clicks and execute a callback function
+
+Requires App versions 14.8 or higher
+
+#### React example
+
+```tsx
+React.useEffect(() => {
+    const unsubscribe = onNavigationBarIconClicked(({iconId}) => {
```

Comment:
shouldn't this just be "id" ?

Reply:
I just followed the Tech Spec

@yamal-alm ?

Reply:
just "id" is ok for me 👍